doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ===============
 
+v4.8.0
```

Comment:
4.7.0 is still not released, can you add these changes there instead of creating a new release?

Reply:
Fixed in the commit: [a326b26](https://github.com/lsst-ts/ts_idl/pull/123/commits/a326b265d9d4dc8923966ae4abfa5551896c98f4). Thanks!